fission/src/mirabuf/MirabufSceneObject.ts

```diff
@@ -986,3 +989,29 @@ export class RigidNodeAssociate extends BodyAssociate {
 }
 
 export default MirabufSceneObject
+
+export class MirabufObjectChangeEvent extends Event {
+    private static _eventKey = "MirabufObjectChange"
+    private _obj: MirabufSceneObject | null
+
+    private constructor(obj: MirabufSceneObject | null) {
+        super(MirabufObjectChangeEvent._eventKey)
+        this._obj = obj
+    }
+
+    public static addEventListener(cb: (object: MirabufSceneObject | null) => void): () => void {
+        const listener = (event: Event) => {
+            if (event instanceof MirabufObjectChangeEvent) {
+                cb(event._obj)
+            } else {
+                cb(null)
+            }
```

Comment:
It would be a better pattern to assign to an intermediate variable with a ternary, then call the function with that variable, similar to what you did [later in this diff](https://github.com/Autodesk/synthesis/pull/1272/files#diff-eb5ba54c9d67d833dc023240994bb072a4dfa657d3c4c03a7f3fcf0352c43b1dR61-R62). This is just a nitpick though, feel free to ignore it if you don't care.

Reply:
will be made obsolete anyway by #1275 so I don't think it matters